lib/client-api/src/client_api.ts

```diff
@@ -50,6 +50,21 @@ export const addParameters = (parameters: Parameters, deprecationWarning = true)
   singleton.addParameters(parameters);
 };
 
+const addLoaderDeprecationWarning = deprecate(
+  () => {},
+  `\`addLoader\` is deprecated, and will be removed in Storybook 7.0.
+Instead, use \`export const loaders = [];\` in your \`preview.js\`.
+Read more at https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-addparameters-and-adddecorator).`
+);
```

Comment:
We don't need this we aren't exporting `addLoader`

Reply:
We are exporting it because that's what the virtual module code calls